docs/docs/lib/node.mdx

```diff
@@ -21,20 +21,86 @@ npm install --save @growthbook/growthbook
 
 ## Quick Usage
 
-GrowthBook instances are scoped to a single incoming request. The easiest way to do this is with Middleware:
+First, create and instantiate a singleton GrowthBook client. You can do this as part of your server startup script.
+
+```js
+const client = new GrowthBookMultiUser({
+  apiHost: "https://cdn.growthbook.io",
+  clientKey: "sdk-abc123"
+});
+await client.init({timeout: 3000});
+```
+
+Then, you can evaluate a feature anywhere in your app by providing a feature key and a `userContext`.
+
+```js
+// User context with attributes that feature flags can use for targeting
+const userContext = {
+  attributes: {
+    id: "123",
+  }
+}
+
+// Boolean on/off flags
+if (client.isOn("my-feature", userContext)) {
+  console.log("My feature is on!");
+}
+
+// String, Number, or JSON flags
+const value = client.getFeatureValue("my-string-feature", "fallback", userContext);
+console.log(value);
+```
+
+### Express Middleware
+
+Instead of constantly defining and passing `userContext` around your app, we recommend creating a "scoped" instance of GrowthBook in a middleware instead.
```

Comment:
It'd be helpful to have another sentence on why you need to create a scoped instance/what it is.